docker-compose.dev.yml

```diff
@@ -152,6 +152,31 @@ services:
     environment:
       PORT: "8080"
       CI: "true"
+      DANGEROUSLY_DISABLE_HOST_CHECK: "true"
```

Comment:
- [ ] I had to use this because otherwise I would not be able to make http calls from the `e2e` container.

Reply:
And as a last resort, we could override the the `allowedHosts` option in Webpack Dev Server: https://craco.js.org/docs/configuration/devserver/. As this isn’t exposed directly as a config option, it requires a hacky workaround using craco and will likely break in the future, so if something else works, I’d prefer that.

Reply:
Ah so setting `ALEPH_UI_HOST` is enough, great to know that! I thought about the docker-compose override file as well, but I'd like to avoid it if it's not necessary. Thanks!

Reply:
Perhaps this is https://github.com/facebook/create-react-app/issues/12304 ?

Reply:
Meh, I doubled down and went with the docker compose override for now 🤷

Reply:
Yeah, I played around with it for a bit yesterday trying to find a solution that doesn’t require ignoring the host check or hacky workarounds to adjust the configuration of the underlying Webpack dev server, but without success :/